cirrus/server/tests/test_main.py

```diff
@@ -49,7 +50,26 @@ def test_read_root(client):
     assert response.json() == {"Hello": "World"}
 
 
-def test_get_features_with_required_field(client):
+@pytest.mark.parametrize(
+    "url, expected_output",
+    [
+        (
+            "/v1/features/",
+            {"example-feature": {"enabled": False, "something": "wicked"}},
+        ),
+        (
+            "/v2/features/",
+            {
+                "Features": {
+                    "example-feature": {"enabled": False, "something": "wicked"},
+                    # return default features
+                },
+                "Enrollments": [],
+            },
+        ),
```

Comment:
I get why you'd fold them together like this but I think we should keep the v1 and v2 tests completely isolated from one another as a way to say 'these changes do not impact the design or behaviour of the v1 endpoints' so we know that by making these changes we aren't impacting the v1 endpoint in any way.